packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1049,10 +1050,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.validateSummaryHeuristicConfiguration(this.summaryConfiguration);
         }
 
+        const compressionOptions = this.mc.config.getBoolean("Fluid.ContainerRuntime.DisableCompression") === true ?
```

Comment:
overall, we should strive to have only one reference for a feature gate in code. Here, specifically, it's trivial given both places we read are in same function. It will save bundle size and make any future changes simpler.